index.html

```diff
@@ -74,83 +74,14 @@ <h1>Hey there. I’m Stephen. <br/><span class="span">I’m a software developer
     </div>
     
     <section class="work">
+
         <h1 class="work-title">My Recent Works</h1>
         <i class="fa-solid fa-horizontal-rule"></i>
 
         <div class="card-layout">
-            <div class="card">
-                <div class="card-body">
-                    <h3 class="card-title">Multi-Post Stories Gain+Glory</h3>
-                    <ul>
-                        <li>Ruby on rails</li>
-                        <li>css</li>
-                        <li>JavaScript</li>
-                        <li>html</li>
-                    </ul>
-                    <a href="" class="see-more">See Project</a>
-                </div>
-            </div>
-            <div class="card">
```

Comment:
- [ ] I would highly recommend that you ensure that your pop-up fits within the viewport height of any screen by ensuring that it is 100vh. This can be achieved by reducing the height of the images

![image](https://user-images.githubusercontent.com/98453979/216026440-5836dcff-b69e-4006-bdea-40ce336a8cbd.png)